recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
```

Comment:
I'd say that's the correct solution actually.  The source of truth for the version is the git repo.  However, if you've downloaded raw source that's not a git repo, you need to augment that source with the version, which you do by writing it to `conda/.version`.